core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -238,6 +240,12 @@ public class GenericContainer<SELF extends GenericContainer<SELF>>
 
     private boolean hostAccessible = false;
 
+    private final ServiceLoader<CreateContainerCmdCustomizer> globalCreateContainerCustomizers = ServiceLoader.load(
+        CreateContainerCmdCustomizer.class
+    );
+
+    private final Set<CreateContainerCmdCustomizer> localCreateContainerCustomizers = new LinkedHashSet<>();
```

Comment:
Given that there is no API to remove entries, WDYT about having just one `Set` and initializing it with `ServiceLoader.load(...)` values?